iosApp/fastlane/Fastfile

```diff
@@ -76,6 +76,39 @@ platform :ios do
       impact = (framework_size_mb / app_size_mb) * 100  # Calculate the impact of the framework on the total size
       File.open("sizes.txt", "a") { |file| file.write("#{framework_name}: #{'%.1f' % framework_size_mb} MB, Impact: #{'%.1f' % impact}%\n") }
     end
+  end
+
+  desc "Generate simulator build"
+  lane :simulator_app do
+    match(
+      type: "development",
+      app_identifier: ["com.passkeys.twilio.sampleCode"]
+    )
+    settings_to_override = {
+      :BUNDLE_IDENTIFIER => "com.passkeys.twilio.sampleCode",
+      :PROVISIONING_PROFILE_SPECIFIER => ENV["VERIFY_PASSKEYS_DEMO_PROVISIONING_PROFILE"],
+      :DEVELOPMENT_TEAM => ENV["VERIFY_PASSKEYS_DEMO_DEVELOPMENT_TEAM"],
+    }
+    gym(
+      scheme: "iosApp",
+      project: "iosApp.xcodeproj",
+      silent: true,
+      skip_package_ipa: true,
+      skip_archive: true,
+      export_method: "development",
+      xcargs: settings_to_override,
+      destination: "generic/platform=iOS Simulator",
+      configuration: "Debug",
+      derived_data_path: "simulator_build/",
+    )
+  end
+
+  desc "Deploy Sample App to Firebase"
+  lane :deploy_demo_app do
+    # Get the current directory
+    puts "Printing files"
+    puts `ls -l -p ../`
+    puts `ls -l -p ../IPAs/`
```

Comment:
Are these prints for testing? Can they be removed?

Reply:
Fixed, thanks